app/js/lib/server/settings.js

```diff
@@ -90,30 +87,6 @@ export default class Settings extends EventDispatcher {
     return API_VERSION;
   }
 
-  get pushEndpoint() {
-    return this[p.values].get(settings.PUSH_ENDPOINT);
-  }
-
-  set pushEndpoint(value) {
-    this[p.updateSetting](settings.PUSH_ENDPOINT, value);
-  }
-
-  get pushPubKey() {
-    return this[p.values].get(settings.PUSH_PUB_KEY);
-  }
-
-  set pushPubKey(value) {
-    this[p.updateSetting](settings.PUSH_PUB_KEY, value);
-  }
-
-  get pushAuth() {
-    return this[p.values].get(settings.PUSH_AUTH);
-  }
-
-  set pushAuth(value) {
-    this[p.updateSetting](settings.PUSH_AUTH, value);
-  }
-
```

Comment:
Yes, IIRC, we did that for debug so that we can inspect these values in the local storage.